src/main/java/frc/robot/RobotContainer.java

```diff
@@ -252,24 +261,42 @@ private Command intakeCommand() {
   }
 
   private Command shooterOuttake() {
-
     return Commands.parallel(
             triggerCommandFactory.getEjectCommand(), intakeCommandFactory.reverseIntakeCommand())
         .asProxy();
   }
 
   private void configDriverFeedBack() {
     new Trigger(trapElvCommandFactory.getSourceBreak())
-        .and(OI.getTrigger(OI.Driver.intake))
```

Comment:
We are checking the OI.Driver.intake trigger twice, once to do the intake and another to trigger rumble.

We should see if we can include this in the intake command, or in the command group that is attached to the Intake trigger.

Also, we have a signaling subsystem that has rumble.  Should we use that instead?

Reply:
to both points: possible, out of scope of this PR